### PR TITLE
Compat to work on 0.6 through 1.0

### PR DESCRIPTION
**Pull request by iamed2**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,16 +5,22 @@ os:
...
 matrix:
   allow_failures:
     - julia: nightly
   fast_finish: true
 notifications:
   email: false
-script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("CloudWatchLogs"); Pkg.test("CloudWatchLogs"; coverage=true)'
+codecov: true
```
Comment (ararslan) [inline .travis.yml]: :sunglasses:

File: .travis.yml [hunk 2]
```diff
@@ -5,16 +5,22 @@ os:
...
 after_success:
-  - julia -e 'cd(Pkg.dir("CloudWatchLogs")); Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
-  - julia -e 'Pkg.add("Documenter"); cd(Pkg.dir("CloudWatchLogs")); include(joinpath("docs", "make.jl"))'
+  - |
+    julia -e 'VERSION >= v"0.7.0-DEV.3656" && import Pkg
```
Comment (ararslan) [inline .travis.yml]: Seems like this step could be combined with the one below it. Nbd though.

**Comment by iamed2** [comment]
We're on the same page :)

**Comment by ararslan** [comment]
...and now they aren't getting that error at all. That's odd.

**Pull request merged**